assets/js/modules/ads/components/dashboard/PartnerAdsWidget.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * PartnerAdsWidget component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import whenActive from '../../../../util/when-active';
+import whenScopesGranted from '../../../../util/whenScopesGranted';
+import { ADWORDS_SCOPE, MODULES_ADS } from '../../datastore/constants';
+import PAXEmbeddedApp from '../common/PAXEmbeddedApp';
+import { AdBlockerWarning } from '../common';
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { CORE_WIDGETS } from '../../../../googlesitekit/widgets/datastore/constants';
+const { useSelect } = Data;
+
+function PartnerAdsWidget( { WidgetNull, Widget } ) {
```

Comment:
We use PAX in the component names, and functions everywhere, maybe it would be better if we rename this component to `PAXWidget` ?

Reply:
Yeah, that's fair, I'll make the change 👍🏻

Reply:
Actually, the thing about "PAX" alone is that it obscures stuff for new developers. We should avoid acronyms-only, but `PartnerAdsPAXWidget` works fine too. I'm gonna go with that just so it isn't less clear but is also consistent 🙂